src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -132,6 +132,13 @@ ReportAie::writeReport(const xrt_core::device* /*_pDevice*/,
 
 	if(tile.second.find("dma") != tile.second.not_found()) {
           _output << boost::format("    %s:\n") % "DMA";
+
+	  _output << boost::format("        %s:\n") % "FIFO";
+
+	  auto& fifo_node  = tile.second.get_child("dma.fifo_info");
+          _output << fmt16("%s") % "Counter0" % fifo_node.get<std::string>("fifo_counter0");

```

Comment:
This code should be refactored to that it can report 1, 3 or more counters.  Yes, I understand that this currently the case, but there is too much hard-coding here and it needs to be more dynamic.

Reply:
By looking at following spec
http://cervino-doc/r2p19/tile_links/xregdb_me_tile_doc.html#tile_memory___DMA_FIFO_Counter
You can safely assume that, there are two fifo counters.